src/components/list/DraggableList.vue

```diff
@@ -0,0 +1,106 @@
+<script>
+import { mapGetters, mapMutations, mapActions } from 'vuex';
+import draggable from 'vuedraggable';
```

Comment:
you can use here whatever name you want, and I prefer to use the same naming as for our components
DraggableExternal

for example
2 words
first big letter
and my proposition is External for not our components